src/redux/categories/categories.js

```diff
@@ -1,14 +1,31 @@
-const COMPLETED = 'bookstore/categories/COMPLETED';
-
-const categoriesReducer = (state = {}, action) => {
+const STATUs = 'COMPLETED';
+const categories = [
+  'Action',
+  'Adventure',
+  'Biographies and Autobiographies',
+  'Comic Book or Graphic Novel.',
+  'Cookbooks',
+  'Detective and Mystery',
+  'Economy',
+  'Fantasy',
+  'Historical Fiction',
+  'Horror',
+  'Literary Fiction',
+  'Romance',
+  'Short Stories',
+  'Suspense and Thrillers',
+  'Sciences Fiction',
+  'True Crime',
```

Comment:
- [ ] The categories are not showing in the add books, so no need for them right now.